docs/adding-a-page.md

```diff
@@ -98,7 +98,7 @@ export default (
 
 ### Starting the server
 
-Now that the component is setup we can run `npm start dev:search` and navigate to
+Now that the component is setup we can run `yarn dev:search` and navigate to
```

Comment:
Technically this page is very out of date since it still references the search app which is now admin (which we're likely deleting). But c'est la vie!